www/main/sttm-ui/switch/Switch.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+const Switch = ({ title, controlId, className, onToggle, defaultValue = false }) => {
```

Comment:
Is this the same as a toggle?

Reply:
I am not sure I understand. This component is what we would use for all the switches / toggles in our app (like presenter view, in settings etc).